storybook/config.js

```diff
@@ -0,0 +1,18 @@
+import { configure } from '@kadira/storybook';
+import React from 'react';
+import { storiesOf, action } from '@kadira/storybook';
+
+import './storybook.css'
+// for now just simply $ rake asset:precompile && mv public/assets/application-ebf... storybook/application.css
```

Comment:
Is this because some of my components use CSS? What's the way forward here? Move the CSS into something like Radium, or the other way around, use CSS modules or whatever they're called..?